### PR TITLE
Hack#28: Put input field text into a text block for fun personalization

### PR DESCRIPTION
**Pull request by brianndiiri**

### CODE REVIEW

File: webflow-hacks/misc-hacks/28/28.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+document.addEventListener('DOMContentLoaded', function () {
+  // get the element
+  const INPUT_SELECTOR = '[fs-hacks-element="name-input"]';
+  const TEXT_SELECTOR = '[fs-hacks-element="name-text"]';
+  const textElement = document.querySelector<HTMLDivElement>(TEXT_SELECTOR);
+  const inputElement = document.querySelector<HTMLInputElement>(INPUT_SELECTOR);
+  // early return if no element found
+  if (!textElement || !inputElement) return;
+  // on every input field key stroke
+  inputElement.addEventListener('keyup', function () {
```
Comment (alexiglesias93) [inline webflow-hacks/misc-hacks/28/28.ts]: Any reason why you're using `keyup` instead of `input`? I'm curious

Comment (brianndiiri) [reply webflow-hacks/misc-hacks/28/28.ts]: Never thought of `input`

Comment (brianndiiri) [reply webflow-hacks/misc-hacks/28/28.ts]: but wouldn't it be better when the user does copy paste

Comment (brianndiiri) [reply webflow-hacks/misc-hacks/28/28.ts]: changing this to input...

**Review by alexiglesias93 — APPROVED** [review]

**Pull request merged**